libs/build_miniconda3.sh

```diff
@@ -67,7 +67,13 @@ source $prefix/etc/profile.d/conda.sh
 echo "disabling conda auto updates"
 conda config --system --set auto_update_conda False
 echo "install $version of conda"
-conda install -yq conda=$version
+#conda install -yq conda=$version
+conda config --add channels conda-forge
+conda config --set channel_priority strict
+conda create -n regional_workflow -y jinja2 pyyaml f90nml
```

Comment:
I don't think this is acceptable in the stack source code.
It appears that you are creating a conda env called `regional_workflow`.
That should be a recipe.

Reply:
See the other examples on how conda or virtual envs are created for e.g. `ufswm`

Reply:
Updating the description of the PR and the changes introduced with the recent update.